src/magento/magento-marketplace.md

```diff
@@ -1,31 +1,33 @@
 ---
 title: Magento Marketplace
-group: operations
+group: getting-started
 redirect_from:
   - /magento/magento-connect.html
 ---
 
-Magento Marketplace is our next generation application store that offers merchants a curated selection of solutions, and provides qualified developers the tools, platform, and prime location to build a thriving business.
+[Magento Marketplace][1] is our next generation application store that offers merchants a curated selection of solutions, and provides qualified developers the tools, platform, and prime location to build a thriving business.
 
-![]({% link images/images/magento-marketplace.png %})
+The installation process for Marketplace purchases is different for "on premise" installations of Magento Commerce than for Magento installations hosted on [Adobe Commerce Cloud][2].
 
-Go to [Magento Marketplace][1]{:target="_blank"}.
+![]({% link images/images/magento-marketplace.png %}){: .zoom}
+_[Magento Marketplace][1]_
 
-## Find Magento Partners
+## Adobe Commerce Cloud
 
-In the _Admin_ sidebar, click **Find Partners & Extensions**.
+Marketplace extensions can be added to installations of Magento that are hosted on Adobe Commerce Cloud only from a development environment. [Composer][3] is used to add the code to a branch in [GitHub][4], where it is tested in an integration environment and deployed to a staging environment before being pushed to production. To learn more, see [Install, manage, and upgrade extensions][5].
```

Comment:
```suggestion
For installations of Magento that are hosted on Adobe Commerce Cloud, Marketplace extensions can be added only from a development environment. [Composer][3] is used to add the code to a branch in [GitHub][4], where it is tested in an integration environment and deployed to a staging environment before being pushed to production. To learn more, see [Install, manage, and upgrade extensions][5].
```

Reply:
I changed the beginning of this sentence, but removed reference to Composer and GitHub. We don't need to go into that detail here.